### ISSUE TITLE
DirAccess.copy_absolute() doesn't work at exported PCK

### ISSUE THREAD

**Issue by artcattleya** [godotengine/godot#82550]
### Godot version

4.1.1

### System information

Godot v4.1.1.stable.mono - Windows 10.0.17763 - Vulkan (Forward+) - dedicated NVIDIA GeForce GTX 1050 Ti (NVIDIA; 31.0.15.3623) - AMD Ryzen 5 5600X 6-Core Processor (12 Threads)

### Issue description

Copying file from res:// to user:// doesn't work on exported project (return error 7) but working fine in editor.

![247554356-d2626891-1d02-4a64-98c1-fef813b11326](https://github.com/godotengine/godot/assets/73269068/e37c1ed4-38d8-4647-9bf7-706309aa0578)

This was discussed previously at another issue https://github.com/godotengine/godot/issues/74105

### Steps to reproduce

Place code to copy any file from res:// to user:// 
Export project and run.

### Minimal reproduction project

Reproducable project: https://github.com/artcattleya/godot-test  (it was built with version 4.0.3 but also doesn't work at 4.1.1)

**Comment by dalexeev** [comment]
Please ensure that the file is included in the exported project. See [Exporting projects - Resource options](https://docs.godotengine.org/en/stable/tutorials/export/exporting_projects.html#resource-options).

**Comment by artcattleya** [comment]
Yes , i included that file in exported project but it doesn't help

![exportProblem1](https://github.com/godotengine/godot/assets/73269068/142697da-147d-4299-a1ec-e41bedeee3ac)

Also when doing export it shows error (on screenshot above)

**Comment by Zireael07** [comment]
You can't use res:// in exported projects IIRC

**Comment by artcattleya** [comment]
ok, it's working: 
```
var db_name_res = "res://sqlite.db"
var db_name_user = "user://sqlite.db"

#print( DirAccess.copy_absolute(db_name_res, db_name_user))
var dir = DirAccess.open("res://");
if !FileAccess.file_exists(db_name_user):
	dir.copy(db_name_res, db_name_user);
```

**Comment by AThousandShips** [comment]
`res://` is a *relative* path and `copy_absolute` only allows absolute paths, and when exporting a game the bundled resources are not available as absolute paths but only under `res://` AFAIK, it only works in the editor because the resource path is actual files then

**Comment by timothyqiu** [comment]
`res://xxx` itself is absolute. But `copy_absolute()` always use `ProjectSettings.globalize_path()` to convert the input paths to file system paths first.

https://github.com/godotengine/godot/blob/533c616cb86ff7bb940d58ffbbcc1a3eca0aa33d/core/io/dir_access.cpp#L289-L295

In an exported project, `resource_path` is empty, so the `res://` prefix is simply removed and the path becomes a relative path.

https://github.com/godotengine/godot/blob/533c616cb86ff7bb940d58ffbbcc1a3eca0aa33d/core/config/project_settings.cpp#L259-L264

The `globalize_path()` step seems unnecessary.
